### ISSUE TITLE
Twisted has dropped support for Python 2.6

### ISSUE THREAD

**Issue by aapa** [lepinkainen/pyfibot#181]
Latest build [fails in Travis with Python 2.6](https://travis-ci.org/lepinkainen/pyfibot/jobs/95401429), because Twisted no longer supports it.

Excerpt from the log:

```
  Downloading Twisted-15.5.0.tar.bz2 (3.1MB)
    100% |████████████████████████████████| 3.1MB 186kB/s 
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 20, in <module>
      File "/tmp/pip-build-BN6SNQ/twisted/setup.py", line 64, in <module>
        main(sys.argv[1:])
      File "/tmp/pip-build-BN6SNQ/twisted/setup.py", line 45, in main
        from twisted.python.dist import (
      File "./twisted/__init__.py", line 55, in <module>
        _checkRequirements()
      File "./twisted/__init__.py", line 17, in _checkRequirements
        raise ImportError("Twisted requires Python 2.7 or later.")
    ImportError: Twisted requires Python 2.7 or later.

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-BN6SNQ/twisted

The command "pip install -r requirements.txt" failed and exited with 1 during .

Your build has been stopped.
```

[Time to drop support for Python 2.6](https://github.com/lepinkainen/pyfibot/issues/176)

**Comment by aapa** [comment]
Our PyPy though [seems to be 2.5](https://travis-ci.org/lepinkainen/pyfibot/jobs/95401431)?

```
$ python --version
Python 2.7.8 (10f1b29a2bd2, Feb 02 2015, 21:22:43)
[PyPy 2.5.0 with GCC 4.6.3]
```

```
RuntimeError: cryptography 1.0 is not compatible with PyPy < 2.6. Please upgrade PyPy to
use this library.
```

**Comment by aapa** [comment]
Fixed in b4d8716 and #182
